cuqi/samples/_samples.py

```diff
@@ -438,7 +483,7 @@ def plot_autocorrelation(self, variable_indices=None, max_lag=None, combined=Tru
         -------
         axes: matplotlib axes or bokeh figures
         """
-        dim = self.geometry.dim
+        dim = self.geometry_dim
```

Comment:
Again. Does it make sense to plot these things for funvals? And will it work for n-d arrays?

Reply:
for now plotting on function space is restricted to geometries where the function space is represented by a 1D ndarray.